portal-api/api-tests/api-test-users.js

```diff
@@ -253,11 +253,11 @@ const finder = () => {
 };
 
 const apiTestUser = {
-  username: 'api-test-user',
+  username: 'api-tester@ukexportfinance.gov.uk',
   password: 'P@ssword1234',
   firstname: 'API',
   surname: 'Test User',
-  email: 'api-tester@example.com',
+  email: 'api-tester@ukexportfinance.gov.uk',
```

Comment:
Could all of the emails in this file come from constants, similar to `BANK1_CHECKER1.email,` above?